endpoints/api_config.py

```diff
@@ -561,7 +571,7 @@ def __init__(self, name, version, description=None, hostname=None,
       _CheckType(version, basestring, 'version', allow_none=False)
       _CheckType(description, basestring, 'description')
       _CheckType(hostname, basestring, 'hostname')
-      endpoints_util.check_list_type(scopes, basestring, 'scopes')
+      endpoints_util.check_list_type(scopes, (basestring, users_id_token.OAuth2Scope), 'scopes')
```

Comment:
Should the docstring for `scopes` be updated (line 541 - sorry, Github won't let me comment on that line specifically)?

Reply:
Done.